ee/codegen/src/__test__/workflow-value-descriptor-reference/__snapshots__/trigger-attribute-workflow-reference.test.ts.snap

```diff
@@ -1,11 +1,21 @@
 // Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html
 
+exports[`TriggerAttributeWorkflowReference > should generate correct AST for ManualTrigger.config reference 1`] = `
+"ManualTrigger.config
+"
+`;
+
 exports[`TriggerAttributeWorkflowReference > should generate correct AST for ManualTrigger.input reference 1`] = `
 "ManualTrigger.input
 "
 `;
 
-exports[`TriggerAttributeWorkflowReference > should generate correct AST for ManualTrigger.config reference 1`] = `
-"ManualTrigger.config
+exports[`TriggerAttributeWorkflowReference > should generate correct AST for VellumIntegrationTrigger.channel reference 1`] = `
+"SlackMessageTrigger.channel
+"
+`;
+
+exports[`TriggerAttributeWorkflowReference > should generate correct AST for VellumIntegrationTrigger.message reference 1`] = `
+"SlackMessageTrigger.message
```

Comment:
[oos] let's also get some test (and I think general support?) for the code generation of where these triggers get defined